tedana/workflows/tedana.py

```diff
@@ -140,7 +140,7 @@ def _get_parser():
                           type=int,
                           help=('Value passed to repr(mdp.numx_rand.seed()) '
                                 'Set to an integer value for reproducible ICA results; '
-                                'otherwise, set to -1 for varying results across calls.'),
+                                'otherwise, set to 42 for varying results across calls.'),
```

Comment:
I think we actually want to say something like this

```suggestion
                                'otherwise, set to -1 for varying results across calls. '
                                'Default value is 42.'),
```